### PR TITLE
feat: create cart page markup and styles

### PR DESCRIPTION
**Pull request by tetianaman**
https://github.com/user-attachments/assets/d5f78635-04f5-42a0-9cc2-7b97b2e64755

### CODE REVIEW

File: src/components/cart/cart-item/cart-item.module.scss [hunk 1]
```diff
@@ -8,13 +8,13 @@
 
 .itemContent {
     display: grid;
-    grid-template-columns: 75px 1fr;
+    grid-template-columns: 100px 1fr;
```
Comment (tetianaman) [inline src/components/cart/cart-item/cart-item.module.scss:11]: Small fix. I didn't notice that image size in cart item are different in their "narrow" and "wide" versions.

File: src/components/cart/cart-item/cart-item.module.scss [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
     column-gap: 20px;
     align-items: start;
 }
 
 .root:not(:first-child) {
-    border-top: 1px solid rgba(23, 17, 13, 0.3);
+    border-top: 1px solid rgba(23, 17, 13, 0.2);
```
Comment (tetianaman) [inline src/components/cart/cart-item/cart-item.module.scss:17]: I made lines the same in cart page and right panel. Look good.

File: src/styles/common.scss [hunk 3]
```diff
@@ -90,6 +90,19 @@
     color: var(--primary1);
 }
 
+.mutedSecondaryButton {
+    background-color: transparent;
+    color: var(--primary5);
+    border-radius: 5px;
+    font-size: 15px;
+    padding: 13px 16px;
```
Comment (tetianaman) [inline src/styles/common.scss:98]: This is very visually small change. Can be ignored.

File: _codux/ui-kit/buttons/buttons.board.tsx [hunk 4]
```diff
@@ -30,10 +30,16 @@ export default createBoard({
...
                 <Kit.Item>
                     <Variant name="Secondary Button">
-                        <button className="button secondary">Add to Cart</button>
+                        <button className="button">Add to Cart</button>
                     </Variant>
                     <Kit.Description>Secondary Button</Kit.Description>
                 </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Secondary Muted Button">
+                        <button className="button mutedSecondaryButton">Checkout</button>
```
Comment (tetianaman) [inline _codux/ui-kit/buttons/buttons.board.tsx:39]: btw, maybe write some neutral text in the button: "Button", variant name, "Your label"? Or to do it together with other buttons in separate PR?

File: app/routes/cart/route.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+import { CartItem } from '~/components/cart/cart-item/cart-item';
+import styles from './route.module.scss';
```
Comment (alisey) [inline app/routes/cart/route.tsx:2]: Let's put this below JS imports, separated by a line.

**Review by alisey — APPROVED** [review]
The page isn't linked to from anywhere, right? I don't want to accidentally release it with mock data.

**Comment by tetianaman** [comment]
> The page isn't linked to from anywhere, right? I don't want to accidentally release it with mock data.

not linked.

**Pull request merged**